sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/plan/DefaultStep.java

```diff
@@ -143,16 +140,6 @@ public Status getStatus() {
         }
     }
 
-    @Override
-    public List<Element<?>> getChildren() {
-        return Collections.emptyList();
-    }
-
-    @Override
```

Comment:
PlansResource isn't correct anymore.  As in the [restart example](https://github.com/mesosphere/dcos-commons/blob/master/sdk/scheduler/src/main/java/com/mesosphere/sdk/api/PlansResource.java#L159-L162), so should things be interruptable and continuable.

You're comment regarding the error state is correct.  However it is common that a Step not enter the Error state, but continuously loop through Pending, Prepared, Starting, Pending, Prepared, Starting ...  In all cases an end user needs the ability to stop the processing of a Step regardless of its status.